runtime/print.cc

```diff
@@ -130,21 +132,20 @@ class printer : public node_visitor {
   void visit(const constant* c) override { *this << c->value; }
 
   void visit(const let* l) override {
-    *this << "let [";
-    for (const auto& s : l->lets) {
-      *this << s.first << " = " << s.second << "; ";
-    }
-    *this << "] in " << l->body;
+    *this << "let {";
+    print_vector(l->lets);
```

Comment:
do we need/want a separator between the lets? eg `;`

Reply:
The default is a comma. Here's what the output looks like now: 
```
let {
  _ = buffer_min(out, 0),
  _0 = buffer_max(out, 0),
  _1 = buffer_min(out, 2),
  _2 = buffer_max(out, 2)
} in ...
```